tests.py

```diff
@@ -192,9 +192,9 @@ def test_noise_sfts(self):
         )
         max_values_noise_and_signal = np.max(data, axis=0)
         max_freqs_noise_and_signal = freqs[np.argmax(data, axis=0)]
-        self.assertTrue(len(times) == int(np.ceil(self.duration / self.Tsft)))
+        self.assertTrue(len(times) == self.duration / self.Tsft)
         # with signal: all SFTs should peak at same freq
-        self.assertTrue(len(np.unique(max_freqs_noise_and_signal)) == 1)
+        # self.assertTrue(len(np.unique(max_freqs_noise_and_signal)) == 1)
```

Comment:
I don't think agree with this statement @dbkeitel : The whole point of CW is that signals suffer from a frequency modulation, hence this does *not* have to be true in general (It's true we assume `Tsft` such that the signal is "reasonably well contained", but longer time scale modulations can be present).

Reply:
Absolutely correct, this won't be true in general. Only for very short signals without extra (e.g. binary) modulation.

We could check the expected modulation in +- bin counts and require the max to be within that?